scripts/core_build.sh

```diff
@@ -136,6 +136,30 @@ if [[ ! -d ${BUILD_OUTPUT_DIR} ]]; then
   mkdir ${BUILD_OUTPUT_DIR}
 fi
 
+CMAKE_GENERATOR="Unix Makefiles"
+
+# MSYS system
+if [ "$MSYSTEM" == "MINGW64" ] ; then
+  BUILD_UNITTEST=OFF
```

Comment:
I noticed you've turned ut off, and Windows Github action stops at the building.  Please do at least one thing to ensure correctness:

- Turn on ut and run ut in Github action
- Or run static-checks in our `Code Checker matrix` https://github.com/milvus-io/milvus/runs/5415943419?check_suite_focus=true

Reply:
Now, leave the build job for in `Build and test` pipeline, and add `make verifiers` in the `Code Checker` pipeline.